packages/glib.rb

```diff
@@ -13,11 +13,18 @@ class Glib < Package
   depends_on 'python27'
 
   def self.build
-    system "./configure"
+    system "./configure --libdir=#{CREW_LIB_PREFIX}"
     system "make"
   end
 
   def self.install
     system "make", "DESTDIR=#{CREW_DEST_DIR}", "install"
+    FileUtils.cd("#{CREW_DEST_PREFIX}/include") do
+      system "cp -r glib-2.0/* ."
+      system "cp #{CREW_DEST_LIB_PREFIX}/glib-2.0/include/glibconfig.h ."
+      system "rm -rf #{CREW_DEST_LIB_PREFIX}/glib-2.0/"
+      system "rm -rf #{CREW_DEST_LIB_PREFIX}/gio/"
+      system "rm -rf glib-2.0/"
```

Comment:
This moves `glibconfig.h` from the non-standard glib-2.0 directory to the standard `/usr/local/include` location to avoid an error that mentions the file is not found.